Phase1/udemy.html

```diff
@@ -0,0 +1,306 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a076d05399.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="mystyle.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>Udemy</title>
+  </head>
+  <body>
+    <!-- first section -->
+    <header>
+      <nav class="navbar header shadowed">
+        <a href="/" class="logo main">
+          <img
+            src="/images/logo-udemy.svg"
+            alt="Udemy logo"

```

Comment:
all images failed to load since there is an extra `/` in the beginning it should be `src="images/logo-udemy.svg"
`